Improbable/Stdlib/Improbable.Stdlib/Platform/Platform.cs

```diff
@@ -29,9 +27,9 @@ public static class Platform
         private static string ToolbeltConfigDir =>
             Environment.GetEnvironmentVariable("IMPROBABLE_CONFIG_DIR") ?? (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
                 ? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".improbable")
-                : Path.Combine(Environment.GetEnvironmentVariable("HOME"), ".improbable"));
+                : Path.Combine(Environment.GetEnvironmentVariable("HOME") ?? ".", ".improbable"));
```

Comment:
Is using the current directory really the correct thing to do here? If `IMPROBABLE_CONFIG_DIR` and `HOME` are both not set, it feels appropriate to throw imo

Reply:
Unless this mimics the behaviour of toolbelt?